ballerina-test/src/test/java/org/ballerinalang/distribution/lengthValidation/LengthValidator.java

```diff
@@ -0,0 +1,120 @@
+/*
+  ~ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  ~ */
+
+package org.ballerinalang.distribution.lengthValidation;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.NoSuchElementException;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+import java.nio.file.Paths;
+import static org.ballerinalang.distribution.utils.TestUtils.EXAMPLES_DIR;
+import static org.ballerinalang.distribution.utils.TestUtils.EXTENSTIONS_TO_BE_FILTERED_FOR_LINE_CHECKS;
+import static org.ballerinalang.distribution.utils.TestUtils.OUT;
+
+public class LengthValidator {
```

Comment:
Shall we include javadoc comments?  Please include them for public methods as well